service/framework/kvmconfig/framework.go

```diff
@@ -0,0 +1,182 @@
+package kvmconfig
```

Comment:
This I packed to a common package. I didn't want to add `unversioned` suffix. I think we can agree this is the unversioned.

Reply:
I feel like the naming is confusing but we can iterate from here. I think the direction is good.

Reply:
Agreed, but I think something we can get used to. I don't have a better idea ATM.